cirq/ops/common_channels.py

```diff
@@ -187,3 +189,528 @@ def depolarize(p: float) -> DepolarizingChannel:
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __repr__(self) -> str:
+        return 'cirq.amplitude_damping(gamma={!r})'.format(self._gamma)
+
+    def __str__(self) -> str:
+        return 'amplitude_damping(gamma={!r})'.format(self._gamma)
+
+    def _circuit_diagram_info_(
+        self, args: protocols.CircuitDiagramInfoArgs
+    ) -> str:
+        return 'AD({!r})'.format(self._gamma)
+
+
+def amplitude_damping(gamma: float) -> AmplitudeDampingChannel:
+    """
+    Returns an AmplitudeDampingChannel with the given damping rate
+
+    This channel evovles a density matrix via
+            \rho -> M_0 \rho M_0^\dagger + M_1 \rho M_1^\dagger
+
+    With
+        M_0 = \begin{bmatrix} 1 & 0  \\ 0 & \sqrt{1 - \gamma} \end{bmatrix}
+        M_1 = \begin{bmatrix} 0 & \sqrt{\gamma} \\ 0 & 0 \end{bmatrix}
+
+    Args:
+        gamma: the damping rate
+
+    Raises:
+        ValueError is gamma is not a valid probability.
+    """
+    return AmplitudeDampingChannel(gamma)
+
+
+class PhaseDampingChannel(raw_types.Gate):
+    """Phase Damping of a qubit"""
+
+    def __init__(self, gamma) -> None:
+
+        """
+        This channel models phase damping which is the loss of quantum
+        information without the loss of energy
+
+        This channel evovles a density matrix via
+            \rho -> M_0 \rho M_0^\dagger + M_1 \rho M_1^\dagger
+
+        With
+            M_0 = \begin{bmatrix} 1 & 0  \\ 0 & \sqrt{1 - \gamma} \end{bmatrix}
+            M_1 = \begin{bmatrix} 0 & 0 \\ 0 & \sqrt{\gamma} \end{bmatrix}
+
+        Args:
+            gamma: The damping rate
+
+        Raises:
+            ValueError is gamma is not a valid probability.
+        """
+
+        def validate_probability(p, p_str):
+            if p < 0:
+                raise ValueError('{} was less than 0.'.format(p_str))
+            elif p > 1:
+                raise ValueError('{} was greater than 1.'.format(p_str))
+            return p
+
+        self._gamma = validate_probability(gamma, 'gamma')
+
+    def _channel_(self) -> Iterable[np.ndarray]:
+        return (
+            np.array([[1.0, 0.0], [0.0, np.sqrt(1.0 - self._gamma)]]),
+            np.array([[0.0, 0.0], [0.0, np.sqrt(self._gamma)]])
+        )
+
+    def _eq_tuple(self):
+        return (PhaseDampingChannel, self._gamma)
+
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return NotImplemented
+        return self._eq_tuple() == other._eq_tuple()
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __repr__(self) -> str:
+        return 'cirq.phase_damping(gamma={!r})'.format(self._gamma)
+
+    def __str__(self) -> str:
+        return 'phase_damping(gamma={!r})'.format(self._gamma)
+
+    def _circuit_diagram_info_(
+        self, args: protocols.CircuitDiagramInfoArgs
+    ) -> str:
+        return 'PD({!r})'.format(self._gamma)
+
+
+def phase_damping(gamma: float) -> PhaseDampingChannel:
```

Comment:
Rename to `phase_damp`, similar to how we have `cirq.measure` instead of `cirq.measuring`. Method names should be verbs (usually).

Reply:
fixed